changes/0e9736eaa0eef92f2d033cc8e5059bac.yaml

```
@@ -0,0 +1,5 @@
+---
+desc: Fixed epoch timestamp calculation by anchoring to UTC instead of local timezone.
```

Maybe something a bit more targeted since this was just related to the toUTC conversion, like:
```suggestion
desc: Fixed an issue with ``$lib.time.toUTC()`` conversions by anchoring epoch to UTC instead of local timezone.
```
change looks good to me otherwise